verify/v1/verify.yaml

```diff
@@ -504,6 +504,27 @@ paths:
                   - $ref: '#/components/schemas/MccMncMismatchResponse'
                   - $ref: '#/components/schemas/DisposableNumberFalseResponse'
                   - $ref: '#/components/schemas/DisposableNumberTrueResponse'
+              examples:
+                success:
+                  summary: Example of a successful response
+                  value:
+                    requestId: "4457068c-194d-4f72-9550-13b81ec054a9",
```

Comment:
this line is giving error while trying to convert the content of yaml into online swagger UI (same in my local). Is it running in your local?

Reply:
you are right... my mistake.
It is the coma at the end of the values